### ISSUE TITLE
Screen reader users cannot hear the full text when navigating to truncated labels

### ISSUE THREAD

**Issue by leon0leon** [fusioncharts/fusioncharts-dist#53]
As shown in the screenshot below, screen reader users cannot hear the full text when navigating to truncated labels after integrating the [Accessibility Extension](https://www.fusioncharts.com/extensions/accessibility/) provided by FusionCharts official.
![image](https://user-images.githubusercontent.com/123917284/218683748-1753fe78-294d-4424-9e64-32699813a2bf.png)
The truncated text is the same for the visual user as well. But, it really isn't providing enough information. 
People with normal eyesight can hover the mouse over the omitted text and get the complete text from the tooltip, but visually impaired users cannot get the complete text through the sound announced by screen readers.
